CMakeLists.txt

```diff
@@ -579,13 +579,13 @@ if(WITH_SYSTEM_BOOST)
   else()
     set(Boost_USE_STATIC_LIBS ON)
   endif()
-  find_package(Boost 1.61 COMPONENTS ${BOOST_COMPONENTS} REQUIRED)
+  find_package(Boost 1.66 COMPONENTS ${BOOST_COMPONENTS} REQUIRED)
```

Comment:
Excellent.

Reply:
@cbodley need to update `make-dist` accordingly.

Reply:
done, thanks